pkg/provisioner/ironic/dependencies.go

```diff
@@ -61,30 +67,36 @@ func (i *ironicDependenciesChecker) checkIronic() (ready bool, err error) {
 	return ready, err
 }
 
-func (i *ironicDependenciesChecker) checkIronicConductor() (ready bool, err error) {
+func (i *ironicDependenciesChecker) checkIronicConductor() (bool, error) {
+
+	bmcAccess, err := bmc.NewAccessDetails(i.host.Spec.BMC.Address, i.host.Spec.BMC.DisableCertificateVerification)
+	if err != nil {
+		return false, errors.Wrap(err, "failed to parse BMC address information")
+	}
 
 	pager := drivers.ListDrivers(i.client, drivers.ListDriversOpts{
 		Detail: false,
 	})
-	err = pager.Err
+	if pager.Err != nil {
+		return false, pager.Err
+	}
 
+	page, err := pager.AllPages()
 	if err != nil {
-		return ready, err
+		return false, err
+	}
+	drivers, err := drivers.ExtractDrivers(page)
```

Comment:
To avoid confusion I wonder should you rename "drivers" as its also used for the drivers package that is imported

Reply:
Good point, will fix, thanks